packages/venia-ui/lib/util/makeUrl.js

```diff
@@ -21,7 +21,8 @@ const joinUrls = (base, url) =>
 
 const mediaBases = new Map()
     .set('image-product', 'catalog/product/')
-    .set('image-category', 'catalog/category/');
+    .set('image-category', 'catalog/category/')
+    .set('image-swatch', 'attribute/swatch');
```

Comment:
Is this where swatch images are served? I'd have to test this but I don't have a magento instance configured to try it out. @dpatil-magento do you know how to test this?

Reply:
I've confirmed on my local that both image swatch `value` and `thumbnail` are in this path. `value` returns a relative path, while `thumbnail` returns an absolute path (which I think is less fragile to bad config).